connector.go

```diff
@@ -90,7 +91,9 @@ func NewConnector(options ...Option) (*Connector, error) {
 	log.Info().Str("addr", "0.0.0.0:8080").Msg("healthcheck listening on /live and /ready")
 
 	//	Initialize trace provider
-	_, err = monitor.InitTracerProvider(context.TODO(), conf.GetString("trace.host.grpc"), c.id(), c.manifest.Version.String(), string(c.env), conf.GetFloat64("trace.sample.ratio"))
+	ctx, cancel := context.WithTimeout(context.TODO(), conf.GetDuration("trace.host.timeout"))
+	defer cancel()
```

Comment:
have you tested this? this line should trigger cancel after returning the connector.

Reply:
yeah afaik all it does is release child context from memory if it returns before the timeout is reached. correct me if I'm wrong